src/app/route.scss

```diff
@@ -0,0 +1,108 @@
+.route-card {
+  width: 50%;
+  margin-top: 3%;
+  margin-left: 25%;
+  height: 80%;
+  padding: 0;
+  background-color: #5f9ea099;
+  color: white;
+  h1 {
+    text-align: center;
+    font-weight: bold;
+    font-size: 3rem;
+    margin: 0;
+  }
+  em {
+    font-size: 0.9rem;
+  }
+}
+
+.text-page-route {
+  text-align: center;
+  font-size: 1.1rem;
+}
+
+.tech-page-route {
+  text-align: center;
+  font-size: 0.9rem;
+  font-style: italic;
+  font-weight: bold;
+  color: rgba(0, 255, 149, 0.859);
+  text-shadow: rgb(0, 149, 255) 2px 2px 2px;
+}
+
+.publ-text {
+  font-size: 1rem;
+}
+
+.contact-icon {
+  font-size: xx-large;
+  margin-right: 2%;
+  vertical-align: middle;
+  margin-bottom: 10px;
+  color: black;
+}
+
+.nav-links {
+  color: blue;
+  text-decoration: underline;
+  &:hover {
+    color: white;
+    text-decoration: underline;
+  }
+}
+
+@mixin logo-route($prop1, $prop2, $prop3) {
+  font-size: 5rem;
+  color: $prop1;
+  background-color: $prop2;
+  margin-right: 2%;
+  margin-top: $prop3;
+}
+
+.JS-logo-route {
+  @include logo-route(#f7df1e, black, 0);
+}
+
+.HTML-logo-route {
```

Comment:
A little bit of inconsistency in the class naming. in the main page you have a classes like "html-icon", here you rather call them logos. Also html here is UPERCASE and there is all small....

Reply:
See:

https://nextjs.org/docs/pages/building-your-application/styling/css-modules

I think the css modules works in any recent react framework.